sbt-metals/src/sbt-test/sbt-metals/semanticdb/build.sbt

```diff
@@ -30,7 +30,7 @@ lazy val b = project
 lazy val c = project
   .in(file("c"))
   .settings(
-    scalaVersion := "2.12.8", // latest scalameta doesn't publish for ~2.12.7
+    scalaVersion := "2.12.9", // latest scalameta doesn't publish for ~2.12.8
```

Comment:
Don't they? https://mvnrepository.com/artifact/org.scalameta/semanticdb-scalac - 4.5.13 is available for 2.12.8

Reply:
https://github.com/scalameta/metals/pull/4463

Reply:
I rebased on the update PR.